src/main/java/com/google/api/codegen/viewmodel/DynamicLangXApiView.java

```diff
@@ -131,6 +131,9 @@ public boolean hasMissingDefaultOptions() {
   @Nullable
   public abstract String gapicPackageName();
 
+  @Nullable
+  public abstract List<String> operationTypes();
```

Comment:
What's this for?

Reply:
Woops, Theres two implementations of operation in google-cloud-core. I accidentally implemented the wrong one and this is a byproduct. Removing.

Reply:
Fixed.